src/test/scala/uk/gov/nationalarchives/tdr/api/routes/TransferAgreementRouteSpec.scala

```diff
@@ -15,81 +15,148 @@ import uk.gov.nationalarchives.tdr.api.utils.{TestDatabase, FixedUUIDSource, Tes
 
 class TransferAgreementRouteSpec extends AnyFlatSpec with Matchers with TestRequest with TestDatabase  {
 
-  private val addTransferAgreementJsonFilePrefix: String = "json/addtransferagreement_"
+  private val addTransferAgreementNotComplianceJsonFilePrefix: String = "json/addtransferagreementnotcompliance_"
+  private val addTransferAgreementComplianceJsonFilePrefix: String = "json/addtransferagreementcompliance_"
   private val getTransferAgreementJsonFilePrefix: String = "json/gettransferagreement_"
```

Comment:
I know this isn't part of the changes you made and it was like this before but could you get rid of this line please. It isn't used anywhere.